index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```

Comment:
- [Optional] I think it would be better to write a descriptive title related to the project you are working on. 
> Descriptive HTML page titles help users quickly understand a web page’s content.

for example 
```
<title> Hello Microverse </title>
```